charts/pip-frontend/values.stg.template.yaml

```diff
@@ -3,3 +3,4 @@ nodejs:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
+    EXCLUDE_E2E: true
```

Comment:
nooo???

Reply:
we need to exclude by now, tech debt ticket was created

Reply:
my wrong, staging should have it